src/main/webapp/templates/dashboard.html

```diff
@@ -3,14 +3,22 @@
 <div class="jumbotron text-center">
```

Comment:
The new dashboard looks pretty cool, but the "DiscAnno" header is not centered which looks weird.